teachertool/src/services/loggingService.ts

```diff
@@ -0,0 +1,24 @@
+const formatMessageForConsole = (message: string) => {
+    const time = new Date();
+    return `[${time.getHours()}:${time.getMinutes()}:${time.getSeconds()}] ${message}`;
+}
+
+const formatName = (name: string) => {
+    return name.toLowerCase().replace(/ /g, "_");
+}
+
+export const logError = (name: string, details: string) => {
+    pxt.tickEvent("teachertool.error", { name: formatName(name), message: details });
+    console.error(formatMessageForConsole(`${name}: ${details}`));
+}
+
+export const logInfo = (name: string, message: string) => {
+    pxt.tickEvent(`teachertool.${formatName(name)}`, { message: message });
```

Comment:
Just noticed we're sending tick events from here. Logging info to telemetry will be overly chatty. Can we please remove it?

Reply:
I'll remove this in a future change.